src/main/kotlin/de/bund/digitalservice/useid/identification/IdentificationSessionController.kt

```diff
@@ -67,6 +71,9 @@ class IdentificationSessionsController(
         path = ["/{useIdSessionId}/$TCTOKEN_PATH_SUFFIX"],
         produces = [MediaType.APPLICATION_XML_VALUE]
     )
+    @Operation(summary = "Get TC token for this session")
+    @ApiResponse(responseCode = "200")
+    @ApiResponse(responseCode = "404", description = "No corresponding session found for that useIdSessionId", content = [Content()])
```

Comment:
What about adding 500 error for all endpoints? Or is that not part of the API docs since it is not intended?

Reply:
Yeah, was wondering about that as well... I thought, that 500 can always happen, when there is some unexpected error, so I didn't document that explicitly. The other response codes have a special semantics for each request, so it's more helpful.
What do you think? Adding 500 everywhere?

Reply:
Just [a random response in a random thread](https://stackoverflow.com/questions/54238746/should-internal-error-500-be-part-of-swagger-api-documentation) but I kind of agree. So let's **not** document 500.

Reply:
Sorry for commenting it everywhere then.

Reply:
No, it's good to have a second thought about it 👍